api/swagger-spec/oapi-v1.json

```diff
@@ -18214,6 +18217,10 @@
       "$ref": "v1.RollingDeploymentStrategyParams",
       "description": "input to the Rolling deployment strategy"
      },
+     "after": {
+      "$ref": "v1.AfterDeploymentActions",
```

Comment:
We can't change post to an array without breaking backwards compatibility.
PostPost is bad.  Having Post at two levels is confusing.  So we need to
have a name that is invoked after the strategy.  We could call it
PostStrategy, but I'm not sure that's easy to understand for users.

On Tue, Feb 2, 2016 at 1:23 PM, Maciej Szulik <notifications@github.com>
wrote:

> In api/swagger-spec/oapi-v1.json
> <https://github.com/openshift/origin/pull/6925#discussion_r51610860>:
>
> > @@ -18214,6 +18217,10 @@
> >        "$ref": "v1.RollingDeploymentStrategyParams",
> >        "description": "input to the Rolling deployment strategy"
> >       },
> > +     "after": {
> > +      "$ref": "v1.AfterDeploymentActions",
>
> Can't we just stick to Pre/Post names we already have at Strategy levels.
> The only difference is - those will be at Deployment level and not at
> strategy and explain to users how these relate.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6925/files#r51610860>.
>